### ISSUE TITLE
Array Int causes crash on new project

### ISSUE THREAD

**Issue by MichelVGameMaker** [deepnight/ldtk#813]
Hello

I tried to browse but did not find this issue... which seems weird.
- I create a new project, 
- I create an entity layer (leave everything default),
- I create an entity and just add to it an Array of Int
- In the level available in this new project, I place one of this entity, click it, click the + to set the Integer value
Then it crashes. 
Float does the same
String works fine

Please let me know if you need anything. (No hurry on my side)

Stack:
```
LDtk version: 1.2.4-64bits
Syntax error, unrecognized expression: unsupported pseudo: read-only
Error
Error: Syntax error, unrecognized expression: unsupported pseudo: read-only
    at Function.se.error (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:13639)
    at PSEUDO (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:16708)
    at Ee (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:20889)
    at se.compile (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:21783)
    at se.select (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:22998)
    at se (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:7116)
    at Function.se.matches (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:12962)
    at Function.S.filter (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:24772)
    at j (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:24623)
    at S.fn.init.is (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:25232)
```
Processes:
```
#0 App
 |--- #7 page.Editor
       |--- #8 misc.FileWatcher
       |--- #9 display.WorldRender
       |--- #10 display.LevelRender
       |--- #11 display.Camera
       |--- #12 display.Rulers[--]
       |--- #13 tool.SelectionTool[--]
       |--- #14 tool.lt.DoNothing[--]
       |--- #15 WorldTool
       |--- #16 tool.PanView[--]
       |--- #25 tool.lt.EntityTool[--]
       |--- #37 ui.EntityInstanceEditor
       |--- #38 tool.ResizeTool[--]
       |--- #39 display.EntityRender
       |--- #40 display.EntityRender
```

Log:
```
[RENDER]       Rendering world bg...
[USERACTION]   Opened panel #22 ui.modal.panel.EditLayerDefs
[FILE]         Loading HTML template editLayerDefs
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[TIDY]         Fixed layer instance array in data.Level#37f55b30-7820-11ed-9597-85aaa27bedd0 "Level_0" (order mismatch or missing layer instance)
[TIDY]         Added missing layer instance Entities in data.Level#37f55b30-7820-11ed-9597-85aaa27bedd0 "Level_0"
[EVENT]        LayerInstanceSelected
[EVENT]        LayerDefAdded
[RENDER]       Rendered level UI
[RENDER]       Rendering world level data.Level#37f55b30-7820-11ed-9597-85aaa27bedd0 "Level_0"...
[WARNING]      Full level render requested
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[USERACTION]   Opened panel #31 ui.modal.panel.EditEntityDefs
[USERACTION]   Closed panel #22 ui.modal.panel.EditLayerDefs
[FILE]         Loading HTML template editEntityDefs
[FILE]         Loading HTML template fieldDefsForm
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[FILE]         Loading HTML template pivotEditor
[EVENT]        EntityDefAdded
[FILE]         Reading HTML template pivotEditor from cache
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[RENDER]       Rendering world bg...
[EVENT]        FieldDefAdded Integer
[USERACTION]   Closed panel #31 ui.modal.panel.EditEntityDefs
[EVENT]        LayerInstanceEditedByTool
[USERACTION]   EntityTool: mode=Add, in LayerInst#1 "Entities" [Entities]
[FILE]         Loading HTML template entityInstanceEditor
[USERACTION]   Added entity EntityInst "Entity" @16,16
[EVENT]        EntityInstanceAdded 2
[RENDER]       Rendering world bg...
[EVENT]        EntityFieldInstanceChanged 3
[USERACTION]   Changed field: FieldInst "Integer" = 0 [ [null] ]
[GENERAL]      Page started: page.CrashReport()
[FILE]         Loading page template: crashReport from C:/Users/Michel/Desktop/LDTK124/resources/app.asar/assets/tpl/pages/crashReport.html
[ERROR]        Syntax error, unrecognized expression: unsupported pseudo: read-only (Error)
[ERROR]        Error: Syntax error, unrecognized expression: unsupported pseudo: read-only
    at Function.se.error (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:13639)
    at PSEUDO (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:16708)
    at Ee (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:20889)
    at se.compile (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:21783)
    at se.select (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:22998)
    at se (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:7116)
    at Function.se.matches (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:12962)
    at Function.S.filter (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:24772)
    at j (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:24623)
    at S.fn.init.is (C:\Users\Michel\Desktop\LDTK124\resources\app.asar\assets\js\jquery.min.js:2:25232)

[GENERAL]      
#0 App
 |--- #7 page.Editor
       |--- #8 misc.FileWatcher
       |--- #9 display.WorldRender
       |--- #10 display.LevelRender
       |--- #11 display.Camera
       |--- #12 display.Rulers[--]
       |--- #13 tool.SelectionTool[--]
       |--- #14 tool.lt.DoNothing[--]
       |--- #15 WorldTool
       |--- #16 tool.PanView[--]
       |--- #25 tool.lt.EntityTool[--]
       |--- #37 ui.EntityInstanceEditor
       |--- #38 tool.ResizeTool[--]
       |--- #39 display.EntityRender
       |--- #40 display.EntityRender
 |--- #41 page.CrashReport

[FILE]         Cleared all file watches
```

**Comment by deepnight** [comment]
Thanks for reporting, I'll have a look.

**Comment by deepnight** [comment]
Sorry about that one, this will be fixed in 1.2.5

**Comment by MichelVGameMaker** [comment]
Don't worry and thank you very much. No hurry on my side
